### ISSUE TITLE
AZ-700 Timing

### ISSUE THREAD

**Issue by arnoldvilleneuve** [MicrosoftLearning/AZ-700-Designing-and-Implementing-Microsoft-Azure-Networking-Solutions#232]
![AZ-700 Timing](https://github.com/MicrosoftLearning/AZ-700-Designing-and-Implementing-Microsoft-Azure-Networking-Solutions/assets/1082625/490e464e-7800-4bbf-ba40-27ad83e8e294)

There is 24 hours worth of time allocated to labs for a three day course. Given that labs usually take half as long as time allocated let's just say there is 12 hours of labs for a three day course. 

Additionally, the course timing suggests students to lab exercises during the lunch hour. This is a totally unreasonable expectation and simply due to fail. These courses are intense and cover complex topics. Students need a rest and a break from learning so they can recoup their mental capacity to learn more. Microsoft knows that because of its documentation in learning and the MCT/MCE program certification dictums. 

m1 67
m2 49
m3 58
m4 43
m5 37
m6 69
m7 33
m8 26

Total 382

18 hours over three days without labs is 1080 minutes / 382 slides is .3 slides per minute or roughly 1 slide every 3 minutes which is a normal pace. However, there is also 12 hours worth of labs to be done within an 18 hour course. 

The Trainer Handbook states that the Instructor should not go through every slide. Since the students do not receive the slides what is the point of providing slides that a) the Instructor cannot review and b) the students will never receive? 

Students taking this course should have a similar learning experience from one Instructor to the next. Leaving the playing field so wide open with the Training Guide stating that each Instructor should adapt the course to suite their purpose makes no sense and does not achieve a consistent learning experience that helps students properly prepare to be successful at passing the exam. There is an AZ-700 Exam Guide that makes it very clear the topics that need to be properly covered in the course. 

https://learn.microsoft.com/en-us/credentials/certifications/exams/az-700/

Every delivery of the AZ-700 should be fairly consistent regardless of the Instructor delivering it. 

Please reduce the slide count to what is required to help students properly learn the topic and prepare for the exam. 

Please reduce the "required" labs to what is required to help students properly learn the topic and prepare for the exam. 

In other words, please stick to the exam guide and stop throwing everything including the kitchen sink into a MOC course because you can. It does not help Instructors deliver a focused and consistent learning experience. 

Now, I also noticed that there are no outstanding issues for this course. Starting to teach it tomorrow morning. We shall see!
